versions.yaml

```diff
@@ -20,47 +20,36 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 _variables:
-  from9: &from9 gcr.io/google-appengine/debian9
-  gpg: &gpg 573BFD6B3D8FBC641079A6ABABF5BD827BD9BF62
-
 versions:
-  - dir: '1/debian9/1.5'
-    repo: influxdb1
-    templateSubDir: debian9
-    tags:
-      - '1.5.5-debian9'
-      - '1.5-debian9'
-      - '1.5.5'
-      - '1.5'
-    from: *from9
-    packages:
-        influxdb:
-          gpg: *gpg
-          version: 1.5.5
-  - dir: '1/debian9/1.6'
-    repo: influxdb1
-    templateSubDir: debian9
-    tags:
-      - '1.6.6-debian9'
-      - '1.6-debian9'
-      - '1.6.6'
-      - '1.6'
-    from: *from9
-    packages:
-        influxdb:
-          gpg: *gpg
-          version: 1.6.6
   - dir: '1/debian9/1.7'
     repo: influxdb1
-    templateSubDir: debian9
+    templateSubDir: influxdb1
     tags:
       - '1.7.10-debian9'
       - '1.7-debian9'
       - '1.7.10'
       - '1.7'
       - 'latest'
-    from: *from9
+    excludeTests:
+      - tests/functional_tests/influx_smoke_test_for_V2_test.yaml
+    from: gcr.io/google-appengine/debian9
     packages:
         influxdb:
-          gpg: *gpg
+          gpg: 05CE15085FC09D18E99EFB22684A14CF2582E0C5
           version: 1.7.10
+  - dir: '2/debian10/2.0'
+    repo: influxdb2
+    templateSubDir: influxdb2
+    tags:
+      - '2.0.7-debian10'
+      - '2.0-debian10'
+      - '2.0.7'
+      - '2.0'
+      - 'latest'
+    from: gcr.io/google-appengine/debian9
```

Comment:
IMO should be updated to buster in case of influxdb 2

change gcr.io/google-appengine/debian9 -> marketplace.gcr.io/google/debian10

Reply:
done